quesma/main.go

```diff
@@ -209,7 +209,7 @@ func constructQuesma(cfg *config.QuesmaConfiguration, sl clickhouse.TableDiscove
 	if cfg.TransparentProxy {
 		return quesma.NewQuesmaTcpProxy(cfg, quesmaManagementConsole, logChan, false)
 	} else {
-		const quesma_v2 = false
+		const quesma_v2 = true
```

Comment:
💪

Reply:
That's only to check if all tests are passing. It's not right time yet to do the switch, maybe next week. We should all agree :)